tests/parser/features/test_logging.py

```diff
@@ -404,17 +455,37 @@ def test_logging_fails_after_a_global_declaration(assert_tx_failed):
 
 def test_logging_fails_after_a_function_declaration(assert_tx_failed):
     loggy_code = """
-
 @public
 def foo():
     pass
 
 MyLog: __log__({arg1: bytes <= 3})
     """
-    t.s = s
     assert_tx_failed(lambda: get_contract_with_gas_estimation(loggy_code), StructureException)
 
 
+def test_logging_fails_when_number_of_arguments_is_greater_than_declaration(assert_tx_failed):
+    loggy_code = """
+MyLog: __log__({arg1: num})
+
+@public
+def foo():
+    log.MyLog(1, 2)
+"""
+    assert_tx_failed(lambda: get_contract_with_gas_estimation(loggy_code), VariableDeclarationException)
```

Comment:
Does it really matter to use `get_contract_with_gas_estimation()` when it's going to fail anyways?

Unless `get_contract()` isn't imported, then disregard

Reply:
@fubuloubu I think we just wrap everything with `get_contract_with_gas_estimation` now so we have code to test the estimation on :)

Reply:
Then maybe we can rename all to `get_contract`, keeping the gas estimation inside. No since in having two fixtures in that case

Reply:
True, and of we want to switch estimates off we just have parameter for get_contract ;)

Reply:
Yeah `def get_contract(code, check_gas=True)`